### PR TITLE
issue templates, slack link

### PR DESCRIPTION
**Pull request by david-vavra**
- Add issue templates (bug, enhancement)
- drop pull-request-template
- drop nonnecessary parts of contributing guideline
- add Slack reference to README

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,6 @@ tmp
 
 **/*.pos
 **/test_output/
+
+# IDEs
+.idea/
```
Comment (FUSAKLA) [inline .gitignore:11]: This should go to your global `~/.gitignore`

Comment (david-vavra) [reply .gitignore]: Yes, thx.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+about: Create a report to help us improve slo-exporter
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+Please see the dedicated section of [our contributing guideline](https://github.com/seznam/slo-exporter/blob/master/CONTRIBUTING.md#reporting-bugs).
+
+**Describe the bug**
```
Comment (FUSAKLA) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: I'd add a section about the environment the bug happened in. Version of slo-exporter, virtuaization sytem etc maybe?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+FILL ME
+
+**To Reproduce**
```
Comment (FUSAKLA) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
**How to Reproduce**
```

**Comment by FUSAKLA** [comment]
Few nits otherwise :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (david-vavra) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: All this is specified in the referenced document - https://github.com/seznam/slo-exporter/blob/master/CONTRIBUTING.md#reporting-bugs. 

Let's either remove it from the document and specify it here, or leave it as it is now.

Comment (FUSAKLA) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: I'd say the template is enough, and we do not need to reference the contribution guide at all. 
Let's make it easier and complicate it once there is a need for it?

Comment (david-vavra) [reply .github/ISSUE_TEMPLATE/bug_report.md]: https://github.com/seznam/slo-exporter/pull/47/commits/173c9b63a334c1b565df34891349bd0ab91f901d

**Comment by david-vavra** [comment]
Failing test is obviously not connected to this documentation related PR, it's yet another case of the [known flakiness](https://github.com/seznam/slo-exporter/issues/35).

**Pull request merged**